File: jambeth/jAmbeth.Event/src/de/osthus/ambeth/event/store/IEventStoreHandlerExtendable.java
package de.osthus.ambeth.event.store;

public interface IEventStoreHandlerExtendable
{
	void registerEventStoreHandler(IEventStoreHandler eventStoreHandler, Class<?> eventType);

	void unregisterEventStoreHandler(IEventStoreHandler eventStoreHandler, Class<?> eventType);
}

File: ambeth/Ambeth.Event/ambeth/event/store/IEventStoreHandlerExtendable.cs
using System;

namespace De.Osthus.Ambeth.Event.Store
{
    public interface IEventStoreHandlerExtendable
    {
        void RegisterEventStoreHandler(IEventStoreHandler eventStoreHandler, Type eventType);

        void UnregisterEventStoreHandler(IEventStoreHandler eventStoreHandler, Type eventType);
    }
}